platforms/jsl/overlay.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_ICELAKE=y
+CONFIG_INTEL_DMIC=y
+CONFIG_INTEL_SSP=y
+CONFIG_LP_MEMORY_BANKS=1
```

Comment:
These are a bit problematic as these properties are not SW configuration, but HW properties that should be set on Zephyr side. 

But alas, these are the SOF side definitions and they are defined in the "samples/" dir in Zephyr, so I guess better to move them back to SOF now.